SARS-CoV-2-Mpro/consolidate-work-to-pandas.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+"""
+Extract final work values to a single pandas dataframe
+"""
+
+import os
+
+projects = {
+    'complex' : 13418,
+    'solvent' : 13419,
+    }
+
+NUM_EXPECTED_WORK = 4001
+NUM_EXPECTED_WORK = 401
+NUM_EXPECTED_WORK = 41
+
+phases = list(projects.keys())
+
+basepaths = [
+    '/home/server/server2/data/SVR314342810/',
+    ]
```

Comment:
Are these values worth moving to command-line options or configuration in a future version?